csi.proto

```diff
@@ -1020,6 +1023,41 @@ message NodeUnpublishVolumeRequest {
 message NodeUnpublishVolumeResponse {
   // Intentionally empty.
 }
+message NodeGetVolumeStatsRequest {
+  // The ID of the volume. This field is REQUIRED.
+  string volume_id = 1;
+
+  // It can be any valid path where volume was previously published.
+  // It MUST be an absolute path in the root filesystem of
+  // the process serving this request.
+  // This is a REQUIRED field.
+  string volume_path = 2;
+}
+
+message NodeGetVolumeStatsResponse {
+  repeated VolumeUsage usage = 1;
+}
+
+message VolumeUsage {
+  enum Unit {
+    BYTES = 0;
```

Comment:
Nit: Most enums typically have a `UNKNOWN = 0;` Can we add one here too?

Reply:
Good catch @ddebroy!

Reply:
Fixed!

Reply:
thanks .. LGTM